### ISSUE TITLE
`_validate_x509_proxy` thread not alive error

### ISSUE THREAD

**Issue by FaroutYLq** [XENONnT/outsource#44]
This issue started yesterday. Same codes ran well last week.
```
'Thread' object has no attribute 'isAlive'
  File "/home/yuanlq/software/outsource/outsource/Shell.py", line 46, in run
    if thread.isAlive():
  File "/home/yuanlq/software/outsource/outsource/Outsource.py", line 516, in _validate_x509_proxy
    shell.run()
  File "/home/yuanlq/software/outsource/outsource/Outsource.py", line 178, in submit_workflow
    self._validate_x509_proxy()
  File "/home/yuanlq/software/outsource/bin/outsource", line 229, in main
    outsource.submit_workflow()
  File "/home/yuanlq/software/outsource/bin/outsource", line 233, in <module>
    main()
AttributeError: 'Thread' object has no attribute 'isAlive'
```

**Comment by ahiguera-mx** [comment]
It seems that thread's method isAlive() was renamed to is_alive() in python version 3.9

**Comment by ahiguera-mx** [comment]
that might fix the issue

**Comment by FaroutYLq** [comment]
Ah yes I forgot to mark it as resolved. It has been solved in #46. Thanks!
